dlhub_sdk/client.py

```diff
@@ -258,7 +258,7 @@ def run(self, name: str, inputs: Any, parameters: Optional[Dict[str, Any]] = Non
         """Invoke a DLHub servable
 
         Args:
-            name: DLHub name of the servable of the form <user>/<servable_name>
+            identifier: DOI of desired servable or DLHub name of the form <user>/<servable_name>
```

Comment:
as it is, `identifier` can be a name or a DOI. only the name of the variable was changed. if that is a problem in and of itself, though, then of course I can add another input (although that would be a little clunky IMO because the user would have to use it by name)